app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/splash/SplashFragment.kt

```diff
@@ -1,12 +1,41 @@
 package com.bottlerocketstudios.brarchitecture.ui.splash
 
-import com.bottlerocketstudios.brarchitecture.R
-import com.bottlerocketstudios.brarchitecture.databinding.SplashFragmentBinding
-import com.bottlerocketstudios.brarchitecture.ui.BaseDataBindingFragment
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import com.bottlerocketstudios.brarchitecture.ui.BaseFragment
+import com.bottlerocketstudios.compose.resources.ArchitectureDemoTheme
+import com.bottlerocketstudios.compose.splash.SplashScreen
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
-class SplashFragment : BaseDataBindingFragment<SplashFragmentViewModel, SplashFragmentBinding>() {
+class SplashFragment : BaseFragment<SplashFragmentViewModel>() {
     override val fragmentViewModel: SplashFragmentViewModel by viewModel()
 
-    override fun getLayoutRes(): Int = R.layout.splash_fragment
```

Comment:
@ianslinker-br Has the splash xml layout file been deleted yet?

Reply:
Not yet, thats actually part of a latter cleanup todo step after the fragments have been removed entirely and replaced by composable call inside a navhost block in the activity